libbeat/docs/step-configure-kibana-endpoint.asciidoc

```diff
@@ -0,0 +1,20 @@
+. If you plan to use the sample Kibana dashboards provided with {beatname_uc},
+configure the Kibana endpoint:
++
+[source,yaml]
+----------------------------------------------------------------------
+setup.kibana:
+  host: "localhost:5601"
+----------------------------------------------------------------------
++
+--
+Where `host` is the hostname and port of the machine where Kibana is running,
```

Comment:
The Kibana configuration is the same as the Elasticsearch output configuration (https://www.elastic.co/guide/en/beats/filebeat/6.x/elasticsearch-output.html#hosts-option).
The `host` can be an `URL` or `IP:PORT`.  For example: http://192.15.3.2, 192:15.3.2:5601, http://192.15.3.2:5601/path. If no port is specified, then 5601 is used.

Note: 
When a node is defined as an IP:PORT, the scheme and path are taken from the `protocol` and `path` config options.